src/components/ProductPage/ProductDataUI.tsx

```diff
@@ -50,6 +54,17 @@ const ProductDataUI: React.FC<ProductDataUIProps> = ({
     if (value) setReferral(value)
   }, [value, setReferral])
 
+  const navReducer = (nav: number, component: IndexComponent): number => {
+    return nav + parseFloat(component.totalPriceUsd)
+  }
+
+  const getNav = () => {
+    let acc = 0
```

Comment:
`acc` doesn't feel super descriptive/useful here. I would prefer to either remove it (put the 0 inline) or rename it